examples/celery-rabbitmq/rabbitmq-service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1beta3
+kind: Service
+metadata:
+  creationTimestamp: null
```

Comment:
You can delete this. Not required. We want the examples to be easy to understand, so its better to omit empty fields.